packages/terra-data-grid/src/DataGrid.module.scss

```diff
@@ -9,13 +9,9 @@
     width: 100%;
   }
 
-  .data-grid {
+  table {
     // Style for the grid itself
-    border-collapse: separate;
-    border-spacing: 0;
-    table-layout: fixed;
     user-select: none;
-    width: 1px;
 
     :focus {
       outline: var(--terra-data-grid-cell-focus-outline, 3px dashed #000);
```

Comment:
Is this style already handled by the table?  Does the DataGrid have a component that would require us to apply this style separately?